### ISSUE TITLE
regression: windows-latest visual studio image reverted (?) to VS 2019 16.6 - should be 16.7?

### ISSUE THREAD

**Issue by oising** [microsoft/azure-pipelines-agent#3075]
## Agent Version and Platform
Version of your agent? `2.172.0`

OS of the machine running the agent? `windows-latest`

## Azure DevOps Type and Version
`dev.azure.com`

If dev.azure.com, what is your organization name?
`https://dev.azure.com/HQCdevops`

## What's not working?
We are required to use Visual Studio task to build our solution because we have SQL projects. We are targeting core 3.1 for several months now, but since you dropped support for 2.2 and 3.0 (no idea if this is related), the VS build task has been failing and so has nuget restore:

```
##[error]The nuget command failed with exit code(1) and error(d:\a\1\s\foobar\foobar.csproj : error :
Version 3.1.401 of the .NET Core SDK requires at least version 16.7.0 of MSBuild. The current available version
 of MSBuild is 16.6.0.22303. Change the .NET Core SDK specified in global.json to an older version that 
requires the MSBuild version currently available.
```

MSBuild resolve log:

```
C:\hostedtoolcache\windows\NuGet\5.6.0\x64\nuget.exe sources Add -NonInteractive -Name NuGetOrg -Source https://api.nuget.org/v3/index.json -ConfigFile d:\a\1\Nuget\tempNuGet_12939.config
Package source with Name: NuGetOrg added successfully.
C:\hostedtoolcache\windows\NuGet\5.6.0\x64\nuget.exe restore d:\a\1\s\foo.sln -Verbosity Detailed -NonInteractive -ConfigFile d:\a\1\Nuget\tempNuGet_12939.config
NuGet Version: 5.6.0.6591
MSBuild auto-detection: using msbuild version '16.6.0.22303' from 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\bin'. Use option -MSBuildVersion to force nuget to use a specific version of MSBuild.
MSBuild P2P timeout [ms]: 192000
```

TL;DR -- dotnet core 3.1 requires msbuild 16.7 (visual studio 2019 servicing 16.7) 

We are not using `global.json`.

## Agent and Worker's Diagnostic Logs
n/a.

**Comment by mjroghelia** [comment]
This is an issue with the software included on the hosted machines, not the agent. You'll want to open an issue at https://github.com/actions/virtual-environments
